DFE-Digital/Dfe.Academies.External
# Removed the Print button partial

* the partial was never used and was written 2y ago
* the assets that were being loaded also referenced missing fonts which caused 4x 404s on every page load
* there was a lot of duplicated CSS that clashed with the gov.uk frontend

## Type of change


## Steps to test this PR

1. Build the app using this PR as the base
2. Load the web page in your browser
3. Ensure CSS/Fonts etc are working as expected
